lightyear/src/netconnection/mod.rs

```diff
@@ -0,0 +1,12 @@
+pub(crate) mod client;
+
+pub(crate) mod server;
+
+pub(crate) mod steam;
+
+pub(crate) mod netcode;
+
+/// The client id from a connect token, must be unique for each client.
+///
+/// Note that this is not the same as the [`ClientId`], which is used by the server to identify clients.
+pub type ClientId = u64;
```

Comment:
Ideally if this could possibly name collide with another type it should be a newtype eg `pub struct ClientId(u64)` so the compiler will appropriate reject it if you use the wrong one.